### ISSUE TITLE
Clicking "Paid" doesn't always update.

### ISSUE THREAD

**Issue by Serinus1** [paxswill/evesrp#75]
On clicking "mark paid" entry will turn green (and mark paid button disabled) for about half a second then return to blue.

Clicking a second time will update and give an error message of:

Paid is not a valid status to change to from Paid (valid options: [<evaluating>, <approved>])

Using Chrome Version 41.0.2272.118 m on Brave's instance.

**Comment by paxswill** [comment]
Yeah, there's a race condition that I *think* I fixed in the current master branch. Right now I'm blocked on another project and figuring out how to write actual tests for the UI portions of the code.
